GoogleForCreators/web-stories-wp
# Dashboard: Implement redesign of the settings page

## Context

- We are implementing the new design system through the dashboard

## Summary

Updates the settings page to match the [new designs](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=467%3A13745).

## Relevant Technical Choices

n/a

## To-do



## User-facing changes

||Before|After|
|---|---|---|
|Scrolling + clicking|![before](https://user-images.githubusercontent.com/22185279/106835553-c0087d80-6654-11eb-93e0-1f6b48211b6b.gif)|![settings-page](https://user-images.githubusercontent.com/22185279/106834331-75860180-6652-11eb-9177-6d558dececc3.gif)|
|Keyboard|![before-tabbing](https://user-images.githubusercontent.com/22185279/106835538-bb43c980-6654-11eb-8cf4-e98891455be6.gif)|![tabbing](https://user-images.githubusercontent.com/22185279/106834322-70c14d80-6652-11eb-9922-c74bedbe16f8.gif)|
|LTR|![ltr](https://user-images.githubusercontent.com/22185279/106835796-30af9a00-6655-11eb-927d-d4a214c33a21.gif)|![rtl](https://user-images.githubusercontent.com/22185279/106835716-12499e80-6655-11eb-8bb3-3bc2466f0658.gif)|



## Testing Instructions



1. Go to editor settings
2. Interact with page to make sure everything still works the same. This should only be visual changes.

### QA






This PR can be tested by following these steps:


1. Go to editor settings
2. Interact with page to make sure everything still works the same. This should only be visual changes.


### UAT






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #4969
